tools/create-image.sh

```diff
@@ -38,6 +95,19 @@ ssh-keygen -f $RELEASE.id_rsa -t rsa -N ''
 sudo mkdir -p $DIR/root/.ssh/
 cat $RELEASE.id_rsa.pub | sudo tee $DIR/root/.ssh/authorized_keys
 
+if [ "$TRINITY" = true ];then
+    sudo chroot stretch /bin/bash -c "mkdir -p ~; cd ~/; wget https://github.com/kernelslacker/trinity/archive/v1.5.tar.gz -O trinity-1.5.tar.gz; tar -xf trinity-1.5.tar.gz"
```

Comment:
wget over https does not work for me under root:
```
Connecting to github.com (github.com)|140.82.118.4|:443... connected.
ERROR: The certificate of 'github.com' is not trusted.
ERROR: The certificate of 'github.com' hasn't got a known issuer.
tar: This does not look like a tar archive
```

Please download it without sudo and then copy.